src/components/ScholarshipAmountField.jsx

```diff
@@ -5,13 +5,17 @@ import {
   RadioGroup,
   FormControl,
   FormControlLabel,
+  FormHelperText,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import PropTypes from 'prop-types';
 import AmountType from '../types/AmountType';
 import AmountTextField from './AmountTextField';
 
 const useStyles = makeStyles({
+  formControlStyle: {
+    paddingTop: 20,
```

Comment:
Use the theme's spacing:
```suggestion
const useStyles = makeStyles((theme) => {
  formControlStyle: {
    paddingTop: theme.spacing(2),
```
I believe the default spacing unit is 8px so this'll give a padding of 16px. Either 1 or 2 units should be fine.